src/main/resources/testchipip/csrc/cospike_impl.cc

```diff
@@ -26,27 +27,27 @@ bool spike_loadarch_done = false;
 #if __has_include ("mm.h")
 #define COSPIKE_SIMDRAM
 #include "mm.h"
-extern std::vector<std::map<long long int, backing_data_t>> backing_mem_data;
+extern std::vector<std::map<unsigned long long int, backing_data_t>> backing_mem_data;
 #endif
 #endif
 
-#define BOOT_ADDR_BASE (0x1000)
-#define CLINT_BASE (0x2000000)
-#define CLINT_SIZE (0x10000)
-#define UART_BASE (0x10020000)
-#define UART_SIZE (0x1000)
-#define PLIC_BASE (0xc000000)
-#define PLIC_SIZE (0x4000000)
+#define _DEFAULT_BOOT_ADDR (0x80000000)
```

Comment:
`_`'ed names to avoid re-definition of constants coming from `riscv/*.h` headers.